doc/radosgw/s3_objects_dedup.rst

```diff
@@ -54,6 +58,14 @@ the underlying media storing the objects (SSD/HDD) since the bucket indices are
 virtually always stored on a fast medium (SSD with heavy memory
 caching).
 
+The admin can throttle the estimate process by setting a limit to the number of
+bucket-index reads per-second (each read brings 1000 object entries) using:
+
+$ radosgw-admin dedup throttle --max-bucket-index_ops=<count>
+
+A typical system performs about 100 bucket-index reads per second (i.e. 100,000 object entries).
```

Comment:
```suggestion
For example, if your system performs about 100 bucket-index reads per second (i.e. 100,000 object entries).
```

Reply:
actually, I think the number 100 is a good rule of thumb for the number of bucket-index reads per-second and not only an example